tests/integration/ceph_base_file_test.go

```diff
@@ -287,6 +287,28 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	assertNoErrorUnlessNotFound(s, err)
 }
 
+func runPreserveFsOnDeleteTest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
+	logger.Infof("Preserve Fs On Delete Smoke Test")
+
+	fsName := "testfs"
+	activeCount := 2
+	createFilesystem(helper, k8sh, s, namespace, fsName, activeCount)
```

Comment:
Instead of creating a new filesystem in this test, I'd suggest patching an existing test based on a flag. The smoke suite could patch it, and other test suites wouldn't patch it, so we get the validation of both scenarios with different test suites. (In the spirit of keeping the integration tests more efficient)